docs/src/features.md

```diff
@@ -16,7 +16,7 @@ The following sections explain the main features of the optimization model based
 
 ## [Flexible Connection of Energy Assets](@id flex-asset-connection)
 
-The representation of the energy system in *TulipaEnergyModel.jl* is based on [Graph Theory](https://www.britannica.com/topic/graph-theory), which deals with the connection between vertices by edges. This representation provides a more flexible framework to model *energy assets* in the system as *vertices*, and to model *flows* between energy assets as *edges*. In addition, it reduces the model size. For instance, connecting assets directly to each other, without having a node in between, allows us to reduce the number of variables and constraints to represent different configurations. For instance, it is becoming more and more common to have hybrid assets like `storage + renewable` (e.g., battery + solar), `electrolyzer + renewable` (e.g., electrolyzer + wind), or `renewable + hydro` (e.g., solar + hydro) that are located in the same site and share a common connection point to the grid.
+The representation of the energy system in *TulipaEnergyModel.jl* is based on [Graph Theory](https://en.wikipedia.org/wiki/Graph_theory), which deals with the connection between vertices by edges. This representation provides a more flexible framework to model *energy assets* in the system as *vertices*, and to model *flows* between energy assets as *edges*. In addition, it reduces the model size. For instance, connecting assets directly to each other, without having a node in between, allows us to reduce the number of variables and constraints to represent different configurations. For instance, it is becoming more and more common to have hybrid assets like `storage + renewable` (e.g., battery + solar), `electrolyzer + renewable` (e.g., electrolyzer + wind), or `renewable + hydro` (e.g., solar + hydro) that are located in the same site and share a common connection point to the grid.
```

Comment:
I was trying to avoid references to Wikipedia, but I guess in this case, it is fine, the article there has several references to the topic.